jquery.floatThead.js

```diff
@@ -420,8 +420,8 @@
           }
           $table.css(layoutFixed);
           $floatTable.css(layoutFixed);
+          $header.after($newHeader);
```

Comment:
For some reason

```
$tbody.before($newHeader);
```

Was not working at all in this two table scenario. I could not figure it out, so I tried this (which should be the same outcome) and it worked.